WordPress/Classes/ViewRelated/Tools/SettingsTextViewController.m

```diff
@@ -50,10 +50,30 @@ - (instancetype)initWithText:(NSString *)text placeholder:(NSString *)placeholde
         _text = text;
         _placeholder = placeholder;
         _hint = hint;
+
+        [self configureInstance];
     }
     return self;
 }
 
+- (instancetype)initWithCoder:(NSCoder *)aDecoder
+{
+    self = [super initWithCoder:aDecoder];
+    if (self) {
+        [self configureInstance];
+    }
+
+    return self;
+}
+
+- (void)configureInstance
+{
+    self.autocorrectionType = UITextAutocorrectionTypeDefault;
+    self.shouldNotifyValue = YES;
+    self.validatesInput = YES;
+}
```

Comment:
Maybe use `_ivar = ` for these instead of incidentally hitting a setter on init.